packages/xstate-fsm/src/index.ts

```diff
@@ -81,7 +81,7 @@ export function createMachine<
   TEvent extends EventObject = EventObject,
   TState extends Typestate<TContext> = any
 >(
-  fsmConfig: StateMachine.Config<TContext, TEvent>,
+  fsmConfig: StateMachine.Config<TContext, TEvent, TState>,
```

Comment:
Just to open a discussion: XState core always uses `<Context, State, Event>` so I'm not sure if @davidkpiano and @Andarist would be okay with the `xstate-fsm` API to differ. In this case, it would obviously be a breaking change to decide in favor of consistency by having the type arguments in the same order as within the core.

Reply:
`TState` is already the third argument in the `State` and `Machine` interfaces in `fsm`, so this PR follows those patterns. Since `TState` defaults to `any` this shouldn't be a breaking change.

Reply:
@CodingDive Moving forward in V5, the generic arguments will always follow `<TContext, TEvent, ...>` in that order.